web/src/components/input/CustomAutoCompleteInput.js

```diff
@@ -1,21 +1,93 @@
-import React from 'react';
+import React, {useReducer} from 'react';
 import {CircularProgress, TextField, Typography} from '@mui/material';
 import Autocomplete from '@mui/material/Autocomplete';
 import {PropTypes} from 'prop-types';
+import { makeStyles } from '@mui/styles';
 
-const CustomAutoCompleteInput = ({label, field, options, onChange, formData, errors}) => {
-  const error = errors?.find((f) => f.property === field);
+export const ERROR_TYPE = {NORMAL: 0, WARNING: 1, ERROR: 2};
+
+const CustomAutoCompleteInput = ({label, field, options, onChange, formData, errors, warnLevelOnNewValue = ERROR_TYPE.NORMAL}) => {
+
+  const useStyles = value =>
```

Comment:
Move the makeStyles outside of the component since it will be called every time the it's rendered.

Reply:
Moved, please wait for rebuild to be completed.